src/planter/planter.service.ts

```
@@ -7,33 +7,56 @@ export class PlanterService {
     }
   }
 
-  async getUserPlantFromPlantId(userPlantId: number, userId: number) {
+  // 모든 식물 정보를 가져오는 함수
+  async getAllPlants() {
     try {
-      return await this.prisma.user_plants.findUnique({
+      return await this.prisma.plants.findMany();
+    } catch (error) {
+      console.log(error);
+      throw new BadRequestException("NO_DATA_FOUND")
```

ESLint

[error] Replace `"NO_DATA_FOUND")` with `'NO_DATA_FOUND');`

(prettier/prettier)